book/modules.md

```diff
@@ -450,17 +454,20 @@ def complete-git-branch [] {
     # ... code to list git branches
 }
 ```
+
 4. Add `export module git.nu` to `mod.nu`
 5. Add the parent of the `completions` directory to your NU_LIB_DIRS inside `env.nu`
+
 ```nu
 $env.NU_LIB_DIRS = [
     ...
     $nu.default-config-dir
 ]
 ```
+
 6. import the completions to Nushell in your `config.nu`
-`use completions *`
-Now you've set up a directory where you can put your completion files and you should have some Git completions the next time you start Nushell
+   `use completions *`
+   Now you've set up a directory where you can put your completion files and you should have some Git completions the next time you start Nushell
 
 > **Note**
```

Comment:
Let's try to avoid using quote-blocks for callouts.  Quote-blocks have certain meaning for screen-readers, among other reasons.  Use a `::: tip Note` or something similar here also. Thanks!